src/shared/Map.ts

```diff
@@ -35,3 +35,20 @@ export type Landmark = z.TypeOf<typeof zLandmark>;
 export const zLandmarkScore = z.number().int().min(1).max(MAX_LANDMARK_SCORE);
 
 export type LandmarkScore = z.TypeOf<typeof zLandmarkScore>;
+
+export const zLandmarkScoreLog = z.object({
```

Comment:
`zLandmarkScore`. "log" means an array of scores.

Reply:
it would conflict with the `const zLandmarkScore = z.number().int().min(1).max(MAX_LANDMARK_SCORE)` above, should it be renamed to `zScore` and `zLandmarkScore`

Reply:
I see. How about renaming this data type and also the table to LandmarkAssessment?

Also in a separate PR, add an assessorId field to the model (and this type), so that we will support assessors who are not the current user.